app/controllers/home_controller.rb

```diff
@@ -0,0 +1,10 @@
+class HomeController < ApplicationController
```

Comment:
Controller'in adini ornekteki gibi `PagesController` yapsan daha iyi olabilirdi ama degistirmene gerek yok.